### PR TITLE
installation: fix invenio_base imports

### PR DESCRIPTION
**Pull request by SamiHiltunen**
- FIX Adds missing `invenio_base` dependency and amends past
  upgrade recipes following its separation into standalone package.

Signed-off-by: Sami Hiltunen sami.mikael.hiltunen@cern.ch

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [x] update copyright year in touched files

**Comment by SamiHiltunen** [comment]
Copyrights updated.

File: requirements-devel.txt [hunk 1]
```diff
@@ -28,4 +28,5 @@
 #     -e git+git://github.com/mitsuhiko/werkzeug.git#egg=Werkzeug
 #     -e git+git://github.com/mitsuhiko/jinja2.git#egg=Jinja2
```
Comment (jirikuncar) [inline requirements-devel.txt]: please remove this comment and add invenio-accounts and invenio-groups

**Comment by SamiHiltunen** [comment]
Removed TODO and added the dependencies.

**Pull request merged**